pig_the_dice.py

```diff
@@ -1 +1,59 @@
 # pig the dice
+
+import random as rand
+
+num = int(input('player num: '))
+player = {str(p):0 for p in range(1, num+1)}
+player['com'] = 0
+winner = ''
+
+while(True):
+
+    for i in player.keys():
+        score = 0
+
+        while(True):
+            print(f'{i} players turn.')
+            if i == 'com':
+                for roll in range(rand.randrange(0,10)):
+                    n = rand.randrange(1,7)
+                    if n==1:
+                        player[i] = 0
+                        break
+                    else:
+                        score += n
+                        player[i] += score
+                        if player[i] >= 100:
```

Comment:
Increase condition of winning 100 to 150

Reply:
yes sir